### PR TITLE
Redirect to prod domain

### PR DESCRIPTION
**Pull request by james-otten**
Redirect (302) to new k8s prod1

Docs: https://nginx.org/en/docs/http/ngx_http_rewrite_module.html

### CODE REVIEW

File: nginx/app.conf [hunk 1]
```diff
@@ -5,12 +5,13 @@ server {
     access_log /var/log/nginx/access.log;
     error_log /var/log/nginx/error.log debug;
 
-    location = /favicon.ico { access_log off; log_not_found off; }
-    location /static/ {
-        root /var/www/html;
-    }
+    rewrite ^/(.*)$ https://db.mesh.nycmesh.net/$1 redirect;
```
Comment (Andrew-Dickinson) [inline nginx/app.conf:8]: Does this work for `.mesh`?

**Review by Andrew-Dickinson — APPROVED** [review]

File: nginx/app.conf (hunk 1 above)
Comment (james-otten) [reply nginx/app.conf:8]: Does not take domain into consideration

**Comment by Andrew-Dickinson** [comment]
Do we need to do extra work to redirect the other services? Do we need to redirect the other services?

**Comment by james-otten** [comment]
> Do we need to do extra work to redirect the other services? Do we need to redirect the other services?

I wasn't planning on it.

**Comment by Andrew-Dickinson** [comment]
> > Do we need to do extra work to redirect the other services? Do we need to redirect the other services?
> 
> I wasn't planning on it.

Feels like we should at least redirect meshforms since it’s user-facing. And at that point we should probably just redirect *.grandsvc? Redirects aren’t that hard right?

**Comment by james-otten** [comment]
This is happening differently, Andrew is writing a config for apache on grand box to redirect all relevant services.

**Pull request closed without merging**